src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/SupportController.java

```diff
@@ -60,10 +62,10 @@ public SupportController(
     public void resendStaffNotifications(
         @PathVariable("referenceNumber") final String referenceNumber,
         @PathVariable("event") final String event,
-        @RequestHeader(value = HttpHeaders.AUTHORIZATION, required = false) final String authorisation
+        @RequestHeader(value = HttpHeaders.AUTHORIZATION, required = true) final String authorisation
```

Comment:
why make this change? are all documents being uploaded to document management?

Reply:
reverted